### PR TITLE
Fix for Step Slug symmetry

### PR DESCRIPTION
**Pull request by sebworks**
Fixes symmetry issues with Step Slug #363 
 
## Changes
 
- Updated 'static/css/misc.less' to fix issues with padding and size.

## Review
 
- @jimmynotjim  
- @anselmbradford 
- @ajbush 
- @duelj 

## Preview
 
screen shot 2015-03-25 at 8 53 19 am

## Notes
- Removed the adjustment to the font-size of the circle for symmetry purposes. Wasn't able to adjust font-size without adding extra markup. 

- Will require Design sign-off

### CODE REVIEW

File: static/css/misc.less [hunk 1]
```diff
@@ -1030,24 +1030,24 @@
...
 .char-icon {
     &:before {
         display: inline-block;
         box-sizing: border-box;
-        width: 1.5em;
-        height: 1.5em;
-        padding: 0.25em 0;
+        width: @size;
+        height: @size;
+        padding: 0;
```
Comment (Scotchester) [inline static/css/misc.less]: I think you can remove the padding property entirely, now.

**Comment by Scotchester** [comment]
If the change in font size is good with our designers, this is a good solution. Too much browser rounding inconsistency with the previous approach.

**Comment by duelj** [comment]
I am on board with the preview above, it's looking good and I know those fake number minicons are proving to be a pain. Curious is FEWDs would rather have designers make custom number minicons down the road so this doesn't have to happen every time we want to use those numbers? Worth the time you guys think?

**Comment by sebworks** [comment]
@duelj , Im unsure because this approach covers (most of these seem unnecessary though) :

    &__1:before { content: '1'; }
    &__2:before { content: '2'; }
    &__3:before { content: '3'; }
    &__4:before { content: '4'; }
    &__5:before { content: '5'; }
    &__6:before { content: '6'; }
    &__7:before { content: '7'; }
    &__8:before { content: '8'; }
    &__9:before { content: '9'; }
    &__0:before { content: '0'; }
    &__A:before { content: 'A'; }
    &__B:before { content: 'B'; }
    &__C:before { content: 'C'; }
    &__D:before { content: 'D'; }
    &__E:before { content: 'E'; }
    &__F:before { content: 'F'; }
    &__G:before { content: 'G'; }
    &__H:before { content: 'H'; }
    &__I:before { content: 'I'; }
    &__J:before { content: 'J'; }
    &__K:before { content: 'K'; }
    &__L:before { content: 'L'; }
    &__M:before { content: 'M'; }
    &__N:before { content: 'N'; }
    &__O:before { content: 'O'; }
    &__P:before { content: 'P'; }
    &__Q:before { content: 'Q'; }
    &__R:before { content: 'R'; }
    &__S:before { content: 'S'; }
    &__T:before { content: 'T'; }
    &__U:before { content: 'U'; }
    &__V:before { content: 'V'; }
    &__W:before { content: 'W'; }
    &__X:before { content: 'X'; }
    &__Y:before { content: 'Y'; }
    &__Z:before { content: 'Z'; }
    &__a:before { content: 'a'; }
    &__b:before { content: 'b'; }
    &__c:before { content: 'c'; }
    &__d:before { content: 'd'; }
    &__e:before { content: 'e'; }
    &__f:before { content: 'f'; }
    &__g:before { content: 'g'; }
    &__h:before { content: 'h'; }
    &__i:before { content: 'i'; }
    &__j:before { content: 'j'; }
    &__k:before { content: 'k'; }
    &__l:before { content: 'l'; }
    &__m:before { content: 'm'; }
    &__n:before { content: 'n'; }
    &__o:before { content: 'o'; }
    &__p:before { content: 'p'; }
    &__q:before { content: 'q'; }
    &__r:before { content: 'r'; }
    &__s:before { content: 's'; }
    &__t:before { content: 't'; }
    &__u:before { content: 'u'; }
    &__v:before { content: 'v'; }
    &__w:before { content: 'w'; }
    &__x:before { content: 'x'; }
    &__y:before { content: 'y'; }
    &__z:before { content: 'z'; }

File: static/css/misc.less [hunk 2]
```diff
@@ -1030,24 +1030,23 @@
     - cfgov-misc
 */
 
+
+// Size of cicular background.
+@size: 1.5em;
```
Comment (anselmbradford) [inline static/css/misc.less]: You might give this a more descriptive name, something like `@icon-size__em` maybe?

Comment (anselmbradford) [inline static/css/misc.less]: Or maybe just `@icon-size` ... I'm not sure whether and how units should appear in Less var names. Opinions?

Comment (sebworks) [inline static/css/misc.less]: I like @icon-size because if you change the units you don't have to change the variable name.

Comment (anselmbradford) [inline static/css/misc.less]: Yeah I wondered if we wanted to discourage changing the units though. CF has a mix of both https://github.com/cfpb/cf-core/blob/gh-pages/src/cf-vars.less#L49-L53

**Comment by jimmynotjim** [comment]
Why use modifier classes at all? We could easily use `counter` to automate this.

Something along the lines of:
```
.step-icons {
  counter-reset: step-icons;

}

.step-icons_icon:before {
  content: counter(step-icons);
  counter-increment: step-icons;
}
```

You can even use any of the list-style-types to change the counter characters.

ex:
```
  content: counter(step-icons, upper-latin);
```

**Comment by Scotchester** [comment]
Good call, if we are okay with [not supporting IE7](http://caniuse.com/#search=counter). (Which I am.)

**Comment by jimmynotjim** [comment]
I mean, it's not like it's completely broken without the steps icons. They're more of an enhancement to the labels. I'm totally ok with downgrading IE7 as long as the basic functionality is still there.

**Comment by Scotchester** [comment]
I agree. (Worth noting that the empty circles would still be there, though.)

**Comment by jimmynotjim** [comment]
Hmm, I was thinking it would ignore the `content` prop because it didn't know what to do with the `counter() `value. I might be wrong though, it might sets the `content` just with an empty value. Have you tested `counter` in IE7 before? I admit I haven't.

**Comment by Scotchester** [comment]
Oh, good point. No, I haven't actually tested it. Maybe they won't show up.

**Comment by sebworks** [comment]
ok, I've created a new issue for refactoring Character icons #428.  I'm going to close this PR.

**Pull request merged**